lib/mayaUsd/commands/layerEditorCommand.cpp

```diff
@@ -1051,6 +1060,28 @@ class RefreshSystemLockLayer : public BaseCmd
         }
     }
 
+    void _notifySystemLockIsRefreshed()
+    {
+        PXR_NS::VtDictionary callbackContext;
+        callbackContext["proxyShapePath"] = PXR_NS::VtValue(_proxyShapePath.c_str());
+        PXR_NS::VtDictionary callbackData;
+
+        UsdUfe::UICallback::Ptr dstCallback = UsdUfe::getUICallback(TfToken("onRefreshSystemLock"));
```

Comment:
Maybe check this before creating the first dictionary. I expect that there won't be callbacks for many users.